.github/workflows/pages.yaml

```diff
@@ -0,0 +1,28 @@
+name: GitHub Actions Pages Publish
+on: [push]
+jobs:
+  pages-publish:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out repository code
+        uses: actions/checkout@v3
+      - name: Setup node v16
+        uses: actions/setup-node@v3
+        with:
+          node-version: '16' # 16 is default, but lets make sure
```

Comment:
this should probably be 16.7

Reply:
maybe `check-latest`? https://github.com/actions/setup-node/blob/main/docs/advanced-usage.md#check-latest-version

Reply:
I think a `check-latest` would be appropriate.